src/content/en/fundamentals/engage-and-retain/push-notifications/sending-messages.markdown

```diff
@@ -0,0 +1,95 @@
+---
+layout: shared/narrow
+title: "Sending messages"
+description: "There are two servers involved in sending a message: your server and a third party messaging server. You keep track of who to send messages to. The third party server handles the routing."
+published_on: 2016-07-01
+updated_on: 2016-07-01
+order: 30
+translation_priority: 0
+authors:
+  - josephmedley
+---
+
+<p class="intro"> It's easy to assume that sending messages must involve
+building a complex messaging system on the server. That's not quite right. There
+are actually two servers involved in sending a message: your server and a third
+party messaging server. You keep track of two things: recipients and recipient-
+specific endpoints on the messaging server. The messaging server handles the
+routing. </p>
+
+{% include shared/toc.liquid %}
+
+## Anatomy of a subscription object {#subscription-anatomy}
+
+In the last section we said that a subscription object must be stringified and
```

Comment:
optional - would be good to link "in the last section" to the previous page and the specific header you're talking aboot.